faceUnlocker.py

```diff
@@ -0,0 +1,105 @@
+import cv2
+import numpy as np
+import face_recognition as fc
+import os
+
+
+
+def unlock():
+    image = os.path.join('asset','img','login','')
+
+    files = []
+    for root,file,dirs in os.walk(image):
+        for _dir in dirs:
+            files.append(_dir)
+
+    if len(files) > 1:
+        print("login must have a unique image")
```

Comment:
I think we are trying to tell the user that the folder from which we are taking the image should have a unique image. If yes, please make this message a little bit clear to the user.

Reply:
Added a comment to it's request. 
[Refer Here](https://github.com/ugroot/GROOT/issues/41)